charts/router/templates/router-deployment.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if not .Values.global.experimental_native_ingress }}
```

Comment:
Thanks @krancour, no worries. We do intend to revert this change at some point in the future in favor of the cleaner tags & conditionals approach.